test-reference/versions.json

```diff
@@ -1,4 +1,30 @@
 {
+  "ckf-1.9-stable": {
+    "ckf":"1.9/stable",
+    "k8s": {
+      "charmed-kubernetes": {
+        "min-supported": "1.24",
+        "max-supported": "1.30"
+      },
+      "microk8s": {
+        "min-supported": "1.25",
+        "max-supported": "1.30"
+      },
+      "aks": {
+        "min-supported": "1.27",
+        "max-supported": "1.30"
+      },
+      "eks": {
+        "min-supported": "1.24",
+        "max-supported": "1.30"
+      }
+    },
+    "juju": {
+      "current-edge": "3.4/edge",
+      "current-stable": "3.4/stable",
+      "future-beta": "3.6/beta"
```

Comment:
future-edge? Couldn't we use 3.5/edge for now?

Reply:
The future for us will be 3.6, we plan to support that version w/o passing through 3.5.
For 3.6 the only available channel is beta.